System_Information.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+:: Create directory "System Information" if doesn't exist
+echo Create folder: System Information
+IF NOT EXIST "System Information" md "System Information"

```

Comment:
This assumes the current working directory is where this batch file is, but it might not be. I recommend doing this first:

`cd /d %~dp0`

(`/d` means also change the current drive, `%~dp0` expands to the directory of the batch file)